docs/plugins/social_login.rst

```diff
@@ -1,10 +1,105 @@
 .. vale off
 
-Social Login
-#############
+Social Login Guide
+=========================
 
- .. image:: images/twilio-sid-authtoken.png
+With Mautic's Social Login, users can easily sign in via their favorite social platforms like Twitter, Facebook, or LinkedIn. The social login feature automatically pre-fills forms with profile data and updates or creates new contacts in Mautic, streamlining the user experience.
+
+Before You Begin: Setup Requirements
+----------------------------------------
+
+Before enabling social login, make sure you have created social media apps on the platforms you want to integrate:
+
+- `X-Developer <https://developer.twitter.com/en/portal/petition/essential/basic-info/>`_
+- `Facebook <https://developers.facebook.com/products/facebook-login/>`_
+- `LinkedIn <https://www.bing.com/ck/a?!&&p=b2b85466e898e3f3JmltdHM9MTcyODQzMjAwMCZpZ3VpZD0wZmNhOGE5ZC05ODA0LTY0OGYtMjVhYy05ZWQwOTk2MzY1NjYmaW5zaWQ9NTE5Mg&ptn=3&ver=2&hsh=3&fclid=0fca8a9d-9804-648f-25ac-9ed099636566&psq=linkedin+developer+app&u=a1aHR0cHM6Ly9kZXZlbG9wZXIubGlua2VkaW4uY29tLw&ntb=1>`_
+
+Once these apps are set up, you're ready to connect them to Mautic!
+
+Step 1: Authorizing Social Media Plugins
+----------------------------------------
+
+Before you can use social login, each social media plugin needs authorization. Here’s how to do it:
+
+To get started with social login, you’ll first need to authorize each plugin:
+
+1. Copy the Callback URL:
+
+   After creating your social app, the platform will ask for a Callback URL.
+
+   - Go to Mautic's plugin configuration window and copy the Callback URL provided there.
+   - Paste it into the appropriate field in your social app setup.
+
+ .. image:: images/Call_back.png
+    :width: 400
+    :alt: Screenshot of a call back url input field.
+
+2. Add Your API Keys:
+
+   Now that your social app is set up:
+
+   - Copy the API Key (Client Key) and API Secret (Client Secret) from the social platform.
+   - Paste these keys into the relevant fields in the Mautic plugin configuration.
+.. image:: images/API_key.png
     :width: 400
-    :alt: Screenshot of the SID and Auth Token fields
+    :alt: Screenshot of an API Key input field.
+
+3. Authorize the Plugin:
```

Comment:
`3. **Authorize the Plugin**: In the Mautic plugin configuration, click **Authorize**. Ensure the plugin is marked as **Published** by toggling the option to “Yes”. Finally, save your configuration to complete the setup. 

Same comment as "1"
`